doc/reference/std/sugar.md

```diff
@@ -97,27 +171,88 @@ you thus have to pass the receiver as first argument to the method.
 The advantage over `using` is that there is no implicit allocation for
```

Comment:
stale, there is no advantage over using.

Reply:
Should I remove altogether from sugar? What exactly do I remove or keep?

Reply:
yeah, remove.

Reply:
there used to be an older using macro, got removed with the intro of real using.

Reply:
At this point, I'll let this PR get in, and you can remove the code and docs and tests.

Reply:
ok